mpqp/execution/providers/google.py

```diff
@@ -316,13 +316,12 @@ def extract_result_SAMPLE(
 
     data = [
         Sample(
-            bin_str="".join(map(bin, state)),
+            bin_str="".join(map(lambda s: format(s, f'0{nb_qubits}b'), state)),
```

Comment:
Elegant way of formatting.

Reply:
could be even better I think:
```suggestion
            bin_str="".join(map(lambda s: f'{s:0{nb_qubits}b}', state)),
```